Directory.Build.props

```diff
@@ -32,6 +32,12 @@
     <TargetFrameworkMonikerAssemblyAttributesFileClean>true</TargetFrameworkMonikerAssemblyAttributesFileClean>
   </PropertyGroup>
 
+  <PropertyGroup>
+    <!-- The version of XliffTasks pulled in by Arcade is fairly old, so here we specify a newer version. -->
+    <!-- This should be removed when the version in Arcade catches up. -->
+    <XliffTasksVersion>1.0.0-beta.19253.2</XliffTasksVersion>
```

Comment:
Is there a corresponding PR in Arcade that update this? Preferably, we should only make this change in Arcade. Updating Arcade version is automated, right?

Reply:
We're not yet at a place where updates to XliffTasks automatically flow into Arcade, but it is being worked on.